npm-publish-all

```diff
@@ -4,23 +4,31 @@ const path = require("path");
 
 const skipList = ['GlobalStyles', 'node_modules'];
 
+// --cleanup
+const CLEANUP_FIRST = process.argv.includes('--cleanup');
+const UNPUBLISH_FIRST = process.argv.includes('--unpublish-first');
+const UPDATE_CROSS_DEPS = process.argv.includes('--update-crossdeps') || true;
+const PUBLISH = process.argv.includes('--publish') || true;
+const FORCE_UPDATE = process.argv.includes('--force-update') || false;
+const HELP = process.argv.includes('-h') || process.argv.length == 2 || false;
+
 const unpublishAll = (items) => {
     items.forEach((item) => {
         const cwd = `./${item}`;
-        const stats = fs.lstatSync(cwd);
+        fs.lstat(cwd, (err, stats) => {
+            if (!err) {
+                const packageJson = `${cwd}\\package.json`;
+                if (stats.isDirectory() && fs.existsSync(packageJson)) {
+                    const json = JSON.parse(fs.readFileSync(packageJson));
+                    console.log(`Current module is ${json.name}`);
+                    console.log(`Current directory is ${cwd}`);
 
-        if (stats.isDirectory() && isProjectFolder(cwd)) {
-            const packageJson = `${cwd}\\package.json`;
-            if (stats.isDirectory() && fs.existsSync(packageJson)) {
-                const json = JSON.parse(fs.readFileSync(packageJson));
-                console.log(`Current module is ${json.name}`);
-                console.log(`Current directory is ${cwd}`);
-
-                const cmdOptsUnpublish = `npm unpublish ${json.name} --force`;
-                let out = proc.execSync(cmdOptsUnpublish, cwd);
-                console.log(out.toString());
+                    const cmdOptsUnpublish = `npm unpublish ${json.name} --force`;
```

Comment:
Oh, this is risky as it removes all published versions, right? Is this the intention? Maybe the script should only unpublished last version? I assume `getRemoteVersion` world work

Reply:
I introduced a specific commandline arg for this. Now we can decide between unpublish latest version of all. It is useful when we work locally with a local NPM repository.